docker-app/qfieldcloud/core/admin.py

```diff
@@ -48,6 +52,26 @@
 Invitation = get_invitation_model()
 
 
+UserEmailDetails = namedtuple(
```

Comment:
Can we move this just before the `EmailAdressView`, so whenever one has to change something, you do not need to scroll hundreds of lines. I think it is worth it to try to add it as a member of the `EmailAddressAdmin` class (not sure if it will work).